x-pack/plugins/endpoint/public/embeddables/resolver/view/process_event_dot.tsx

```diff
@@ -20,6 +21,7 @@ export const ProcessEventDot = styled(
       className,
       position,
       event,
+      bgColor,
```

Comment:
Why is background color a prop here? It is a constant value right?

Reply:
It's to support what we're doing here: https://github.com/elastic/kibana/pull/57459/files/3a97b436b6532a0a215167b8b49ff7f543174e53#diff-b18117271821fdd985eed1e1c4fd749fR83

It's used to create a "mask" of sorts in the stroke around the shape by sending the background color down. The alternative of making an actual `<mask>` out of the shape seems like an interesting project down the road, but maybe not now.

For now, since we're only supporting one background color to start with it _could_ be removed and imported from the `defs`. Would you prefer that?